deploy/azure.tf

```diff
@@ -0,0 +1,192 @@
+# Terraform configuration file to deploy resouces to Azure.
+
+# Set up this variables in your environment before running Terraform:
+#
+#   export ARM_SUBSCRIPTION_ID=<YOUR SUBSCRIPTION ID>
+#   export ARM_CLIENT_ID=<SERVICE PRINCIPAL ID>
+#   export ARM_CLIENT_SECRET=<SERVICE PRINCIPAL SECRET>
+#   export ARM_TENANT_ID=<ACTIVE DIRECTORY DOMAIN ID>
+#
+# or set up them here in this configuration file:
+#
+#   provider "azurerm" {
+#     subscription_id = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#     client_id       = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#     client_secret   = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#     tenant_id       = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#   }
+#
+# Then run this script with
+# ./terraform.exe plan -var-file=tfvars.json
+
+variable location {
```

Comment:
these needs all to have descriptive comments

Reply:
ok